src/models/things.js

```diff
@@ -64,6 +64,42 @@ const Things = {
     });
   },
 
+  /**
+   * Get a list of Things by their IDs.
+   *
+   * {Array} ids IDs of the list of Things to get.
```

Comment:
As above, it would be better for this to filter on hrefs since thing ids are not guaranteed to correspond to the href in the jwt's scope.